3.9/data-science.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: default
+description: Scalable Graph Analytics and GraphML
+---
+Data Science
+==================
```

Comment:
Most of our content uses [setext](https://spec.commonmark.org/0.30/#setext-heading)-style headings but we should change it to the [ATX style](https://spec.commonmark.org/0.30/#atx-heading) because it's quicker to write and manage when adjusting headline levels. The second-level headlines in this new page are ATX style already, so here it's also a matter of consistency.

```suggestion
# Data Science
```